itests/hive-unit/src/test/java/org/apache/hadoop/hive/ql/txn/compactor/TestCrudCompactorOnTez.java

```diff
@@ -70,11 +77,20 @@
 @SuppressWarnings("deprecation")
 public class TestCrudCompactorOnTez extends CompactorOnTezTest {
 
+  private static final String COMPACTION_QUEUE = "tez_queue";
```

Comment:
Just a suggestion, take it or leave it: change to something less generic like "my_compaction_test_queue" for example.

Reply:
ok :)